doc/cpd/101.rst

```diff
@@ -0,0 +1,150 @@
+CPD-101: Key Storage Encryption
+===============================
+
+Metadata
+--------
+
+* CPD Version: 1
+* Status: Open
+
+Description
+-----------
+Today we are holding keys the same way that secrets are used in some container managers.
+Instead of holding keys in base64 and assuming that the ``Storage`` instance is used
+only for Commissaire, we could encrypt the keys to add another layer of safety.
+
+Rationale
+---------
+The likelihood of having a ``Storage`` system that is used only for Commissaire seems
+low. More than likely the same instance will be used for other applications as well.
+By adding encryption to the keys we mitigate could mitigate access from those with
+access to data dumps and storage system access.
+
+
+Design
+------
+The server(s) would house a GnuPG configuration and keys allowing them to encrypt
+and decrypt data. The servers would either need to have the same key across all
+instances or each server would need a copy of other services public keys.
```

Comment:
I don't think there would be any configuration updates. However, the public/private keys for use would need to be imported into the keyring at ``/etc/commissaire/$PLACEWEDECIDEON``